wireapp/wire-ios-mono
# fix: presenting the invite activity VC - WPB-11994

### Issue

After merging navigation overhaul the presentation of the activity view controller for inviting people to Wire has been broken. (only the case where permission to contacts is denied)

Before:
The connect screen dismissed and a modal or popover was presented from the app's main screen.

Now:
The activity view controller is displayed as a result of tapping the invite button.
Only after the activity view controller is dismissed, the connect UI is dismissed as well.

<img width="340" alt="Screenshot 2024-11-04 at 12 36 17" src="https://github.com/user-attachments/assets/d011b741-0e59-4a33-af65-0951a8149a3b">

### Testing

- Log into a personal account and tap connect/new conversation. (test iPad and iPhone)
- Tap "Invite more people"
- Deny the permission to the contacts
- Tap "Maybe later"
- Observe that the navigation controller pops the last screen and then displays the activity view controller
- Dismiss the activity view controller and observe that the connect UI dismisses.

---

### Checklist


---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
